person.rb

```diff
@@ -0,0 +1,20 @@
+class Person
+  def initialize(age:, name: 'Unknown', parent_permission: true)
+    @id = Random.rand(1..1000)
+    @name = name
+    @age = age
+    @parent_permission = parent_permission
+  end
+  attr_reader :id
+  attr_accessor :name, :age
```

Comment:
[OPTIONAL] 

Well made methods you did over here :ok_hand:, 
However you can make it even more clear in the future by following Ruby devs common practice by :
- [ ] Moving your `attr_reader` , `attr_accessor` right after your class definition